core/ui/src/lib/components/repo/ModifiedFilesCard.svelte

```diff
@@ -316,7 +328,12 @@
 							>Deleted
 						</Tooltip>
 					{/if}
-					<p class="font-bold {getFileTextClass(file)}">{file.path}</p>
+					<p class="font-bold {getFileTextClass(file)}">{getFileDisplayString(file)}</p>
+					{#if file.displayName !== ''}
+						<Tooltip class="w-auto bg-secondary-600 dark:bg-space-800 font-semibold shadow-2xl">
+							{file.path}
+						</Tooltip>
```

Comment:
this tooltip, by nature of being placed inside this component, is going to be constraint to the perimeter of the component itself. so when f11r is small, it'll be totally cut off. 

i think this is shippable as-is, but folks will likely have to increase their f11r size to view the entire tooltip.

Reply:
Yeah I think it's consistent with the rest of the app as well, unfortunately